tests/flow/test_distinct.py

```diff
@@ -0,0 +1,104 @@
+from redis_base import RedisGraphTestBase
+
+class ReturnDistinctFlowTest_1(RedisGraphTestBase):
+    @classmethod
+    def createCommands(cls):
+        return """
+            CREATE (:PARENT {name: 'Stevie'})
```

Comment:
what's the difference between ReturnDistinctFlowTest_1 and ReturnDistinctFlowTest_2?

Reply:
They both basically test the same thing, but the order of records in each result set is different. I included both to document this behavior. We might as well keep a single test.